aten/src/ATen/native/EmbeddingBag.cpp

```diff
@@ -798,7 +798,7 @@ embedding_bag(const Tensor &weight, const Tensor &indices,
     padding_idx = maybe_wrap_dim(padding_idx, weight.size(0));
   }
   std::tuple<Tensor, Tensor, Tensor, Tensor> out;
-  if (!weight.requires_grad()) {
```

Comment:
@zou3519 these might fix some failures on the functorch side

see https://github.com/pytorch/functorch/pull/473